lib/tools/adb-commands.js

```diff
@@ -231,7 +231,6 @@ methods.sendTelnetCommand = async function (command) {
         dataStream += data;
         if (readyRegex.test(data)) {
           res = dataStream.replace(readyRegex, "").trim();
-          res = _.last(res.trim().split('\n'));
```

Comment:
It may need to be more complex, then.

See https://github.com/appium/appium-adb/issues/150